serverless/messaging/reference-content/migration.mdx

```diff
@@ -0,0 +1,173 @@
+  description: Learn about the new version of Messaging and Queuing being released for when SQS and NATS go into General Availability, and how migration to the new version will work.
+content:
+  h1: Understanding Messaging and Queuing GA and Migration
+  paragraph: Learn about the new version of Messaging and Queuing being released for when SQS and NATS go into General Availability, and how migration to the new version will work.
+tags: messaging queuing sqs sns nats beta ga general-availability migration api v1 v2 namespace no-namespace
+dates:
+  validation: 2023-09-25
+  posted: 2023-09-25
+categories:
+  - serverless
+---
+
+## Overview
+
+Soon, a new version of Messaging and Queuing will be released. In this version, the concept of **namespaces** will be removed, to streamline your workflows, offer you a better developer experience, and simplify your management of your Messaging and Queuing resources.
+
+The release of this new version in the Scaleway console will coincide with the [NATS](/serverless/messaging/concepts/#nats) and [SQS](/serverless/messaging/concepts/#sqs) protocols leaving [Public Beta](https://www.scaleway.com/en/betas/) and going into General Availability (GA) This means that the Messaging and Queuing product, when used with these protocols, will become billable. [SNS](/serverless/messaging/concepts/#sns) will remain in Public Beta for now while we continue testing and improving the product.
+
+Your NATS resources will automatically be migrated to the new version of the product. However, for SQS and SNS you will need to migrate, by recreating your resources in the new version. You can do this via the API or the console. This is very important, as 30 days after the new version is released, the old version of the product will be decommissioned and all resources still existing within it will be deleted.
+
+For all protocols, the endpoints of your brokers will change and you will need to update these endpoints in any code, applications or architectures using your message brokers.
+
+Read on to find out more about the new no-namespace version of the product, and how migration will work for the protocols you use.
+
+## The current situation: v1alpha1 and namespaces
+
+Whether you create and manage your NATS, or SQS/SNS namespaces and credentials via our [API](https://www.scaleway.com/en/developers/api/messaging-and-queuing/) and [console](https://console.scaleway.com/), they both point towards the same underlying version of the product: `v1alpha1`, which holds your resources.
+
+In this version of the product, you create one or multiple **namespaces**, setting the **protocol** for each namespace as either **NATS** or **SQS/SNS**. Each namespace gives you a **URL** (or endpoint) representing a Messaging and Queuing broker, and also allows you to create **credentials** to access that particular namespace. When you create messages, queues, streams, topics and other resources, they are scoped to that namespace.
```

Comment:
Normally multiple namespace creation has been deprecated "in the dark silent of the night" so they cannot do that.
Maybe we can just say "you create namespaces" ?